python/tvm/meta_schedule/tune_context.py

```diff
@@ -132,5 +134,108 @@ def __init__(
 
     def initialize(self):
         """Initialize the tuning context"""
-
         _ffi_api.TuneContextInitialize(self)  # type: ignore # pylint: disable=no-member
+
+    def generate_design_space(self, mod: IRModule) -> List[Schedule]:
```

Comment:
I think `mod` here can be omitted (or set as optional) because we use `context.mod` as default.

Reply:
Good catch!! Sorry I overlooked the possibility of taking `mod` directly from `TuneContext`!

Reply:
updated :-)